Dockerfile

```diff
@@ -26,7 +26,6 @@ RUN mkdir fineract
 COPY . fineract
 
 WORKDIR fineract
-RUN sed -i 's/localhost/fineractmysql/' ./fineract-provider/build.gradle
```

Comment:
@conradsp you'll probably want to (manually) test that this doesn't break the container? /Cc @awasum